pkg/hintrunner/core/hint_test.go

```diff
@@ -1115,7 +1115,7 @@ func TestAssertLeFindSmallArc(t *testing.T) {
 		vm.Context.Ap = 0
 		vm.Context.Fp = 0
 		// The addr that the range check pointer will point to
-		addr := vm.Memory.AllocateBuiltinSegment(&builtins.RangeCheck{RANGE_CHECK_N_PARTS: 8})
+		addr := vm.Memory.AllocateBuiltinSegment(&builtins.RangeCheck{RangeCheckNParts: 8})
```

Comment:
Leaving the first letter capital since we are using it to initialize it in this file.